FEXCore/Source/Interface/Core/OpcodeDispatcher/Vector.cpp

```diff
@@ -4451,7 +4456,13 @@ void OpDispatchBuilder::VBLENDPDOp(OpcodeArgs) {
     return;
   }
 
-  OrderedNode *Result = VBLENDOpImpl(*this, DstSize, 8, Src1, Src2, Selector);
+  OrderedNode *Result = VBLENDOpImpl(*this,
+                                     DstSize,
+                                     8,
+                                     Src1,
+                                     Src2,
+                                     LoadAndCacheNamedVectorConstant(DstSize, FEXCore::IR::NamedVectorConstant::NAMED_VECTOR_ZERO),
```

Comment:
Maybe hoist this into its own var so everything can stay compact. Ditto for the others

Reply:
Hoisted.